admin/jsonConfig.json

```diff
@@ -0,0 +1,34 @@
+{
+  "type": "panel",
+  "i18n": true,
+  "items": {
+    "_basicSettings": {
+      "newLine": true,
+      "type": "header",
+      "text": "Please provide your credentials",
+      "sm": 12,
+      "size": 8
+    },
+    "Username": {
+      "newLine": true,
+      "type": "text",
+      "label": "Username",
+      "sm": 4
+    },
+    "Password": {
+      "repeat": false,
+      "visible": false,
+      "sm": 3,
+      "type": "password",
+      "label": "Password"
+    },
+    "intervall": {
+      "sm": 3,
+      "type": "number",
+      "label": "Interval in seconds",
+      "min" : 10,
+      "max" : 300,
```

Comment:
is a max helpful? 300 is only 5 minutes. Maybe we should skip this one ore put it to e.g. 3600

Reply:
good point, not sure about the use cases but indeed updated it to 3600